es/sources_index.go

```diff
@@ -360,9 +361,31 @@ func (index *SourcesIndex) indexSource(mdbSource *mdbmodels.Source, parents []st
 			}
 			authors := authorsByLanguage[i18n.Language]
 			s := append(authors, pathNames...)
-			source.Title = s[len(s)-1]
+			leaf := s[len(s)-1]
+			source.Title = leaf
```

Comment:
Using only the name of the article? Without parents?

Reply:
the parents are in the full_title field